KeenClient/KeenClient.h

```diff
@@ -369,14 +369,31 @@ typedef void (^AnalysisCompletionBlock)(NSData *responseData, NSURLResponse *res
 - (void)runAsyncSavedAnalysis:(NSString *)queryName completionHandler:(AnalysisCompletionBlock)completionHandler;
 
 /**
+ Gets results from a cached dataset query.
+
+ See detailed documentation here: https://keen.io/docs/api/?shell#retrieving-results-from-a-cached-dataset
+ and here: https://keen.io/docs/compute/cached-datasets/
+
+ Results will be grouped by the interval specified in the dataset definition.
+
+ @param datasetName The existing dataset resource name.
+ @param indexValue The required value in the index to retrieve results for.
+ @param timeframe The required timeframe to retrieve results for, which must be a subset of the timeframe specified in the dataset definition.
+ */
+- (void)runAsyncDatasetQuery:(NSString *)datasetName
+                  indexValue:(NSString *)indexValue
```

Comment:
Any particular reason for the term "Value" instead of "By" if the API param is `index_by`?